src/convertToObject.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
Consider adding a check for an empty `sourceString` before proceeding with the split and map operations. This can prevent unnecessary processing and return an empty object immediately if the input is empty.